### ISSUE TITLE
.Net Core compliance

### ISSUE THREAD

**Issue by xgenvn** [Zaid-Ajaj/Cable#1]
Hi,

While Nancy and bridge.net have already supported .NET core, can we upgrade this repository to support it too?
However, having the support under different branch should be good , though.

What do you think?

**Comment by Zaid-Ajaj** [comment]
Hello @xgenvn, It should be possible it is definitely on my todo list, I just didn't have time to properly look into it . PR's are always welcome though :smile: 

Will keep this issue open for the moment.

**Comment by xgenvn** [comment]
It's great that we have same thought. However, I have been trying getting your sample working with .NET core and with Nancy 2.0, there are some changes on NancyModule that makes the service registered fail. I've had to fall back to .net framework.
Nice work, btw. I'll keep an eye on how to make this library being compliance to .Net core.

**Comment by Zaid-Ajaj** [comment]
> I have been trying getting your sample working with .NET core and with Nancy 2.0

Have you tried Nancy 1.4.4 with .NET Core? i.e. 
```
 dotnet add package Nancy --version 1.4.4 
```

**Comment by xgenvn** [comment]
Yes, I'm able to run with latest stable Nancy (1.4.4). I wasn't aware of "pre-release" version.
